docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -37,23 +29,37 @@ these fields as _{feature-vars}_ and _{depvar}_, respectively.
 {feature-vars-cap} are the values that the {depvar} value depends on. There are 
 three different types of {feature-vars} that you can use with our 
 {classification} algorithm: numerical, categorical, and boolean. Arrays are not 
-supported in the {feature-var} fields.
+supported in the {feature-var} fields.  
```

Comment:
> Feature variables

This can perhaps be pursued in a separate PR, but it occurred to me while reading this page that neither our UI nor the API refer to "feature variables". They refer to a "dependent variable" and to included or excluded "analyzed fields". Should we therefore remove this section label and term? Or make it less prominent?

I think this section boils down to something like this: When you create a classification job, you must specify which field contains the classification labels/values/categories that you want to predict (no more than 30). You can also optionally specify which fields to include or exclude from the analysis.  [It might be nice to mention here why you would choose to explicitly include/exclude fields as opposed to letting the analytics do it for you. Why bother? And also maybe mention how you can use the explain API to see which fields are used]

Reply:
@lcawl 
In 860f884, I made feature variables less prominent. I still use the term between brackets though, I felt it won't hurt to name it even if we don't use the term on the UI or in the API docs. If you think we should omit it entirely, we can easily delete it without further complications. Also mentioned why you would choose to include/exclude fields and mentioned the Explain API with a link. I also removed the details of supported fields, but put a link that points to the supported field-types in the PUT DFA API docs (which creates a dependency between this PR and the other creating the link).